src/core/index.ts

```diff
@@ -63,27 +71,106 @@ const approveChange = async (context: Context): Promise<void> => {
  * @param context The request context created by Probot core.
  * @param startTime The timestamp that this run started.
  */
-const createPassingStatus = async (
+export const createPassingStatus = async (
   context: Context,
   startTime: string,
 ): Promise<void> => {
+  await createStatus(
```

Comment:
Codacy found an issue: ['createStatus' was used before it was defined.](https://app.codacy.com/manual/tianhaoz/approveman/pullRequest?prid=6092518)